src/pulse3D/metrics.py

```diff
@@ -565,18 +574,19 @@ def calculate_area_under_curve(
                 value of the width, or the rising or falling coordinates. The final value is either
                 an int representing the width value or a tuple of ints for the x/y coordinates
 
+            baseline_widths: tuple twitch widths to use as baseline metrics
+
         Returns:
             Pandas Series of floats representing area under the curve for each twitch
         """
-        width_percent = 90  # what percent of repolarization to use as the bottom limit for calculating AUC
         estimates_dict: Dict[int, Union[int, float]] = dict()
 
         rising_x_values = coordinate_df["time"]["contraction"].T.to_dict()
         falling_x_values = coordinate_df["time"]["relaxation"].T.to_dict()
 
         for iter_twitch_peak_idx in twitch_indices.keys():
-            start_timepoint = rising_x_values[iter_twitch_peak_idx][width_percent]
-            stop_timepoint = falling_x_values[iter_twitch_peak_idx][width_percent]
+            start_timepoint = rising_x_values[iter_twitch_peak_idx][baseline_widths[0]]
```

Comment:
Should one of these be subtracted from 100?

Reply:
yes, it is above on line 531